src/services/fileStorage/strategies/awsS3.js

```diff
@@ -307,13 +307,13 @@ class AWSS3Strategy extends AbstractFileStorageStrategy {
 				Key: flatFileName,
 				Expires: 60,
 			};
-
-			if (download) {
+			const getBoolean = (value) => value === true || value === 'true';
+			if (getBoolean(download)) {
 				params.ResponseContentDisposition = `attachment; filename = "${localFileName.replace('"', '')}"`;
 			}
-
+			logger.info(params.ResponseContentDisposition);
 			return promisify(awsObject.s3.getSignedUrl.bind(awsObject.s3), awsObject.s3)(action, params);
-		});
+		});	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4849872)